test/test_tree_2d_eulermulti.jl

```diff
@@ -20,11 +20,20 @@ EXAMPLES_DIR = pkgdir(Trixi, "examples", "tree_2d_dgsem")
   end
 
   @trixi_testset "elixir_eulermulti_shock_bubble_shockcapturing_subcell_positivity.jl" begin
+    rm("out/deviations.txt", force=true)
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_eulermulti_shock_bubble_shockcapturing_subcell_positivity.jl"),
       l2   = [81.52845664909304, 2.5455678559421346, 63229.190712645846, 0.19929478404550321, 0.011068604228443425],
       linf = [249.21708417382013, 40.33299887640794, 174205.0118831558, 0.6881458768113586, 0.11274401158173972],
       initial_refinement_level = 3,
-      tspan = (0.0, 0.001))
+      tspan = (0.0, 0.001),
+      output_directory="out")
+      cmd = string(Base.julia_cmd())
+      coverage = occursin("--code-coverage", cmd) && !occursin("--code-coverage=none", cmd)
+      if !coverage
+        lines = readlines("out/deviations.txt")
+        @test lines[1] == "# iter, simu_time, rho1_min, rho2_min"
+        @test startswith(lines[end], "15")
+      end
```

Comment:
I think it is not the best idea to have these very specific coverage checks implemented deep down within a test. I thus suggest to either create a function such as `is_coverage_test()` at a much higher level of the testing infrastructure and just call it here, or to change it to a check that should always work, e.g.,
```suggestion
      lines = readlines("out/deviations.txt")
      @test lines[1] == "# iter, simu_time, rho1_min, rho2_min"
      @test startswith(lines[end], "1")
```

Reply:
Okay. thanks